foreign/node/src/wire/topic/create-topic.command.ts

```diff
@@ -32,7 +32,7 @@ import {
 
 export type CreateTopic = {
   streamId: Id,
-  topicId: number,
+  topicId?: number, // Optional - auto-assigned by server if not provided
```

Comment:
topicId marked "optional" in Type but it is removed from serialize just below - is it optionnal or always server-generated ?